wlib/PreferenceDlg.cpp

```diff
@@ -1964,6 +1951,15 @@ class CPrefOtherDlg : public CDialogImpl<CPrefOtherDlg>,
 		DoChanged(false);
 	}
 
+	void InitComboHelpLocale()
+	{
+		m_comboHelpLocale = GetDlgItem(IDC_COMBO_HELP);
+		m_comboHelpLocale.AddString(_T(""));
+		m_comboHelpLocale.AddString(_T("en"));
```

Comment:
All uppercase "en" and "pt-BR"

Reply:
No - I am keeping them the same casing as what the OS reports when auto detecting.